web/src/main/webapp/app/views/error/tooManyRequests.html

```diff
@@ -0,0 +1,12 @@
+<div>
+	<div class="container-fluid">
+	    <div class="row">
+	    	<div class="overview-content">
+				<div class="col-xs-12 col-sm-10 col-md-10 col-lg-10 alert alert-info">
+					<h1>Du kan bara hämta två rapporter i taget</h1>
```

Comment:
Could we maybe generalise this text? 

The body of the message doesn't talk about absolut limits and it would be good if we can set 1+ calls without needing to change the message.

Reply:
See comment below, I think you can use an already existing endpoint.